great_expectations/cli/cli.py

```diff
@@ -218,82 +218,27 @@ def init(target_directory):
     if not data_source_name: # no datasource was created
         return
 
-    cli_message(
-        """
-========== Profiling ==========
-
-Would you like to profile '{0:s}' to create candidate expectations and documentation?
-
-Please note: Profiling is still a beta feature in Great Expectations.  The current profiler will evaluate the entire 
-data source (without sampling), which may be very time consuming. 
-As a rule of thumb, we recommend starting with data smaller than 100MB.
-
-To learn more about profiling, visit <blue>https://docs.greatexpectations.io/en/latest/guides/profiling.html\
-?utm_source=cli&utm_medium=init&utm_campaign={1:s}</blue>.
-        """.format(data_source_name, __version__.replace(".", "_"))
-    )
-    if click.confirm("Proceed?",
-                     default=True
-                     ):
-        profiling_results = profile_datasource(context, data_source_name)
-        cli_message(
-            """
-========== Data Documentation ==========
-
-To generate documentation from the data you just profiled, the profiling results should be moved from 
-great_expectations/uncommitted (ignored by git) to great_expectations/fixtures.
-
-Before committing, please make sure that this data does not contain sensitive information!
-
-To learn more: <blue>https://docs.greatexpectations.io/en/latest/guides/data_documentation.html\
-?utm_source=cli&utm_medium=init&utm_campaign={0:s}</blue>
-""".format(__version__.replace(".", "_"))
-        )
-        if click.confirm("Move the profiled data and build HTML documentation?",
-                         default=True
-                         ):
-            cli_message("\nMoving files...")
-
-            for profiling_result in profiling_results:
-                data_asset_name = profiling_result[1]['meta']['data_asset_name']
-                expectation_suite_name = profiling_result[1]['meta']['expectation_suite_name']
-                run_id = profiling_result[1]['meta']['run_id']
-                context.move_validation_to_fixtures(
-                    data_asset_name, expectation_suite_name, run_id)
-
-            cli_message("\nDone.")
-
-            cli_message("\nBuilding documentation...")
-            build_documentation(context)
-
-        else:
-            cli_message(
-                "Okay, skipping HTML documentation for now.`."
-            )
-
-    else:
-        cli_message(
-            "Okay, skipping profiling for now. You can always do this "
-            "later by running `great_expectations profile`."
-        )
+    profile_datasource(context, data_source_name)
```

Comment:
All the profiling logic was refactored into profile_datasource and init is calling it.